modules/metering-exposing-the-reporting-api.adoc

```diff
@@ -39,9 +40,10 @@ Be sure to replace the `name=[Report Name]` and `format=[Format]` parameters in
 
 [id="metering-authenticate-using-username-password_{context}"]
 === Authenticate using a username and password
-We are able to do basic authentication using a username and password combination, which is specified in the contents of a htpasswd file. By default, we create a secret containing an empty htpasswd data. You can, however, configure the `reporting-operator.spec.authProxy.htpasswd.data` and `reporting-operator.spec.authProxy.htpasswd.createSecret` keys to use this method.
 
-Once you have specified the above in your MeteringConfig, you can run the following command:
+We are able to do basic authentication using a username and password combination, which is specified in the contents of an htpasswd file. By default, we create a secret containing an empty htpasswd data. You can, however, configure the `reporting-operator.spec.authProxy.htpasswd.data` and `reporting-operator.spec.authProxy.htpasswd.createSecret` keys to use this method.
```

Comment:
I think you may want to substitute "we are able to do basic authentication" with "Metering supports configuring basic authentication using a username and password combination ...", or something alone those lines.

Reply:
@timflannagan Updated with:
> Metering supports configuring basic authentication using a username and password combination, which is specified in the contents of an htpasswd file. By default, a secret containing empty htpasswd data is created.